feat(changelog): functions-added-you-can-now-test 2023-09-27 (#2213)
scaleway/docs-content: changelog/september2023/2023-09-27-functions-added-you-can-now-test.mdx

---

> title: You can now test functions from the console!

Reviewer: ```suggestion
title: You can now test functions from the Scaleway console!
```

status: added
author:

>     fullname: 'Join the #[serverless-functions] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #serverless-functions channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-09-27
category: serverless
product: functions
---

> The new function Test feature allows you to run your code directly from the console using a selection of methods, to make sure it behaves as expected. For more information, refer to the <Link to=https://www.scaleway.com/en/docs/serverless/functions/how-to/test-a-function/>How to test a function</Link> documentation page.

Reviewer: ```suggestion
The new function Test feature allows you to run your code directly from the Scaleway console using a selection of methods, to make sure it behaves as expected. For more information, refer to the [dedicated documentation](/serverless/functions/how-to/test-a-function).
```
